### PR TITLE
feat: Full naming support for route tables rather than just the prefix

### PR DESCRIPTION
**Pull request by syedthameem85**
# Proposed change

The change added the ability for full naming support for route tables rather than just the prefix #103. The existing functionality to specify just prefix is maintained.
## How has these changes been tested?
Using terraform init, plan and apply
### Automated testing

If you're running automated testing for this module, we would love to hear from you, and potentially integrate it to the module standard workflow.

### Manual testing

If no automated testing is run, please ensure that at least the three steps below are passing without any error.

- [ ] Running `terraform apply` on each example provided with this module provisions the intended resource(s) without any errors.
- [ ] Modifying module's *Input Variables* after initial provisioning behaves as intended, i.e: any updateable properties are ameneded without recreation of the resource(s).
- [ ] Running `terraform destroy` on each example provided with this module destroys all the resources created by this module and only the resources created by this module.

## Checklist before submitting your PR

- [ ] My code follows [the style guidelines of this project](../tree/main/docs/codingconventions.adoc)
- [ ] these changes provision new resources:
  - [ ] I have updated the README introduction section (README.adoc)
  - [ ] I have updated the README introduction section (README.md)
- [ ] these changes adds any new variables:
  - [ ] I have updated [docs/terraformoptions.adoc](../tree/main/docs/terraformoptions.adoc) to include each variable
- [ ] I have updated the changelog to include an entry for these changes
- [ ] I have updated all provided examples, including each README file and all applicable code blocks
- [ ] these changes generates no new warnings
- [ ] Any dependent changes have been merged and published in upstream modules

Note: *If you are not an Oracle employee, to contribute to an Oracle-sponsored open-source project, you need to sign the [Oracle Contributor Agreement (OCA)](https://oca.opensource.oracle.com/).*

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -24,6 +24,24 @@ variable "label_prefix" {
   default     = "none"
 }
 
+variable "ig_rt_display_name" {
```
Comment (hyder) [inline variables.tf:27]: There are already variables for the display names e.g. `nat_gateway_display_name`. Please use those instead of defining new ones.

**Review by hyder — CHANGES_REQUESTED** [review]
After making the change related to the existing variable names, can you please also group related variables definitions together in both variables.tf and terraform.tfvars.example e.g.

```
# vcn variable
variable "vcn_cidrs" {
...
}

variable "vcn_dns_label" {
...
}

variable "vcn_name" {
...
}

# nat gateway
variable "create_nat_gateway" {
...
}

variable "nat_gateway_route_rules" {
...
}
```

File: variables.tf (hunk 1 above)
Comment (syedthameem85) [reply variables.tf]: @hyder -This is fixed

**Comment by syedthameem85** [comment]
> After making the change related to the existing variable names, can you please also group related variables definitions together in both variables.tf and terraform.tfvars.example e.g.
> 
> ```
> # vcn variable
> variable "vcn_cidrs" {
> ...
> }
> 
> variable "vcn_dns_label" {
> ...
> }
> 
> variable "vcn_name" {
> ...
> }
> 
> # nat gateway
> variable "create_nat_gateway" {
> ...
> }
> 
> variable "nat_gateway_route_rules" {
> ...
> }
> ```

@hyder- This is fixed

**Comment by hyder** [comment]
Resolves #103

**Review by hyder — APPROVED** [review]
approved

**Pull request merged**